traffic_ops/traffic_ops_golang/dbhelpers/db_helpers.go

```diff
@@ -274,6 +274,46 @@ func GetProfileIDFromName(name string, tx *sql.Tx) (int, bool, error) {
 	return id, true, nil
 }
 
+// GetServerCapabilitiesFromName returns the server's capabilities.
+func GetServerCapabilitiesFromName(name string, tx *sql.Tx) ([]string, error) {
+	caps := []string{}
```

Comment:
this is just a clean up comment but in this function and the function below the code can be cleaned up by using the array type instead of iterating through rows.

```SELECT ARRAY(....)``` and then scanning that into pq.Array. There is a good amount examples of the pattern around the code base.